cli/internal/scm/git_go.go

```diff
@@ -1,3 +1,6 @@
+//go:build go || !rust
+// +build go !rust
```

Comment:
what does this do?

Reply:
There's pragmas that indicate we should turn on the Rust code linked to Go or rely on the old Go code. At some point soon we'll just flip the switch and end up with only the Rust code.